software/apps/libsignpost/xdot.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
```

Comment:
A one liner comment at the top of the .h files with the name of the chip and a URL would be great.